zipkin-cassandra/src/main/scala/com/twitter/zipkin/cassandra/CassandraSpanStoreFactory.scala

```diff
@@ -69,6 +70,7 @@ trait CassandraSpanStoreFactory {self: App =>
       builder.withCredentials(cassandraUsername(), cassandraPassword())
     builder.withRetryPolicy(ZipkinRetryPolicy.INSTANCE)
     builder.withLoadBalancingPolicy(new TokenAwarePolicy(new LatencyAwarePolicy.Builder(new RoundRobinPolicy()).build()))
+    builder.withPoolingOptions(new PoolingOptions().setMaxConnectionsPerHost(HostDistance.REMOTE, cassandraMaxConnections()))
```

Comment:
I think we should be setting `HostDistance.LOCAL`. REMOTE is only for nodes not in the same DC (as determined by the first node you contact), and the driver should prefer LOCAL nodes.